test/web-platform-tests/to-upstream/html/semantics/tabular-data/the-td-element/class-hierarchy.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML>
```

Comment:
This is kind of redundant with the updates done at https://github.com/w3c/web-platform-tests/commit/5b6aa3d108eeb306a326465e87447b53dee68a13 but given that we can't run partial tests I guess it is the best we're going to do...